CMakeLists.txt

```diff
@@ -59,6 +59,9 @@ option(FLATBUFFERS_ENABLE_PCH
 option(FLATBUFFERS_SKIP_MONSTER_EXTRA 
       "Skip generating monster_extra.fbs that contains non-supported numerical\"
       types." OFF)
+option(FLATBUFFERS_OSX_BUILD_UNIVERSAL
+      "Enable the build for multiple architectures on OS X (arm64, x86_64)."
+      ON)
```

Comment:
Shouldn't it be disabled by default? I'm not even sure it makes sense to have this kind of option in a CMakeLists. CMAKE_OSX_ARCHITECTURES should be controlled externally (in a toolchain file).